operator_ui/src/components/TimeAgo.js

```diff
@@ -1,15 +1,10 @@
 import React from 'react'
 import TimeAgoNoTooltip from 'react-time-ago/no-tooltip'
-import StyledTooltip from 'components/Tooltip'
 
 const TimeAgo = ({ children }) => (
-  <StyledTooltip title={children}>
-    <span>
-      <TimeAgoNoTooltip tooltip={false}>
-        {Date.parse(children)}
-      </TimeAgoNoTooltip>
-    </span>
-  </StyledTooltip>
+  <span>
```

Comment:
We still want the tooltip for other areas like dashboard, index etc...

Another reason we need to work on library sharing as this is supported in [explorer](https://github.com/smartcontractkit/chainlink/blob/develop/explorer/client/src/components/TimeAgo.tsx)